test/expectations/selection_ranges/array_literal_oneline.exp.json

```diff
@@ -14,7 +14,7 @@
                 },
                 "end": {
                     "line": 0,
-                    "character": 6
```

Comment:
end columns are off by 1 due to a YARP issue which is being addressed separately.